### PR TITLE
fix dirty repo in drone CI

### PR DESCRIPTION
**Pull request by rosskirkpat**
### Summary
Fixes https://github.com/rancher/rancher/issues/38650

### Occurred changes and/or fixed issues

fixes an issue with previous releases where the version was considered `dirty`

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -13,16 +13,19 @@ steps:
...
-  - mage ci
+    - go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.0
+    - go install github.com/onsi/ginkgo/ginkgo@latest
+    - go get github.com/onsi/gomega/...
+    - cd ..
+    - git clone https://github.com/magefile/mage.git
+    - cd mage
+    - go run bootstrap.go install
+    - cd C:/Drone/src
+    - git status --porcelain --untracked-files=no
```
Comment (aiyengar2) [inline .drone.yml:27]: Should this status print be omitted?

File: .drone.yml [hunk 2]
```diff
@@ -106,17 +109,18 @@ steps:
...
-      - go get -u github.com/onsi/gomega/...
+      - go get github.com/onsi/gomega/...
       - cd ..
       - git clone https://github.com/magefile/mage.git
       - cd mage
       - go run bootstrap.go install
       - cd C:/Drone/src
-      - mage ci
-
+      - mage -v ci
```
Comment (aiyengar2) [inline .drone.yml:123]: Why do we need -v?

File: .drone.yml [hunk 3]
```diff
@@ -13,16 +13,19 @@ steps:
...
+    - go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.0
+    - go install github.com/onsi/ginkgo/ginkgo@latest
+    - go get github.com/onsi/gomega/...
+    - cd ..
+    - git clone https://github.com/magefile/mage.git
+    - cd mage
+    - go run bootstrap.go install
+    - cd C:/Drone/src
+    - git status --porcelain --untracked-files=no
+    - mage -v ci
```
Comment (aiyengar2) [inline .drone.yml:28]: Why do we need -v?

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

**Review by jakefhyde — APPROVED** [review]

File: .drone.yml (hunk 1 above)
Comment (rosskirkpat) [reply .drone.yml:27]: oops, missed this one. removing it

File: .drone.yml (hunk 3 above)
Comment (rosskirkpat) [reply .drone.yml:28]: addressed above - fixing this

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**